File: src/main/java/com/ject/studytrip/dummy/application/dto/DummyMissionsInfo.java
package com.ject.studytrip.dummy.application.dto;

import java.util.List;

public record DummyMissionsInfo(List<DummyMissionInfo> missionInfos) {
    public static DummyMissionsInfo of(List<DummyMissionInfo> missionInfos) {
        return new DummyMissionsInfo(missionInfos);
    }
}

File: src/main/kotlin/com/ject/studytrip/dummy/application/dto/DummyMissionsInfo.kt
package com.ject.studytrip.dummy.application.dto

data class DummyMissionsInfo(
    val missionInfos: List<DummyMissionInfo>,
) {
    companion object {
        @JvmStatic
        fun of(missionInfos: List<DummyMissionInfo>): DummyMissionsInfo = DummyMissionsInfo(missionInfos)
    }
}
